css/trackers.scss

```diff
@@ -0,0 +1,177 @@
+@import "../node_modules/duckduckgo-colors/colors";
+
+body {
+    margin:0;
+    padding:0;
+    font-size:14px;
+    font-weight:bolder;
+}
```

Comment:
We should probably talk about css cascade at some point. Ideally, general rules about the `body` elem should go way higher in the cascade, probably in a `normalize.scss` file and then a `base.scss` file. By the time you get down into something specific like `trackers.scss`, there shouldn't be a need for this here.